pyproject.toml

```diff
@@ -32,13 +32,15 @@ exclude = [
 # are invoked via separate runs of ruff in pre-commit:
 # see our .pre-commit-config.yaml file for details
 exclude = ["**/test_cases/**/*.py"]
-select = [
+external = ["F821", "NQA", "Y"]
```

Comment:
I think you might have dropped a version of this comment accidentally when merging in `main`:

```suggestion
# We still use flake8-pyi and flake8-noqa to check these (see .flake8 config file);
# tell ruff not to flag these as e.g. "unused noqa comments"
external = ["F821", "NQA", "Y"]
```